Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN apk add --update-cache \
 	&& go get -u -v \
 	github.com/tools/godep \
 	github.com/onsi/ginkgo/ginkgo \
-	&& curl -L https://dl.bintray.com/deis/deisci/deis-6c176d2-linux-amd64 -o /usr/local/bin/deis \
+	&& curl -L https://dl.bintray.com/deis/deisci/deis-852b0b0-linux-amd64 -o /usr/local/bin/deis \
```

Comment:
I would say that fixing that is beyond the scope of this PR.  Although we should still figure that out.